programs/create-genesis/main.cpp

```diff
@@ -54,15 +61,25 @@ struct config_reader {
 
 
 void config_reader::set_program_options(options_description& cli) {
-    cli.add_options()
-        ("genesis-info,g", bpo::value<bfs::path>(&info_file)->default_value("genesis-info.json"),
-            "the location of the genesis info file (absolute path or relative to the current directory).")
-        ("output-file,o", bpo::value<bfs::path>(&out_file)->default_value("cyberway-genesis.dat"),
-            "the file to write generic genesis data to (absolute or relative path).")
-        ("ee-output-directory,e", bpo::value<bfs::path>(&ee_directory)->default_value("events-genesis"),
-            "the directory to write Event-Engine genesis data files to (absolute or relative path).")
-        ("help,h", "Print this help message and exit.")
-        ;
+    cli.add_options() (
+        "genesis-info,g", bpo::value<bfs::path>(&info_file)->default_value("genesis-info.json"),
+        "Genesis info file (absolute or relative path)."
+    ) (
+        "state", bpo::value<bfs::path>(&out_genesis_file)->implicit_value("cyberway-genesis.dat")->default_value(""),
+        "If set, create state genesis. If path set, it is output state genesis file (absolute or relative path)."
```

Comment:
what is "state genesis"?, we don't have such term.
Previous description "the file to write generic genesis data to" was clear enough.
And please restore 'o' shortcut.

sidenote: the main purpose of create-genesis utility is to create "consensus" genesis. looks like you name it "state". so the main output is this file. generating "ee-genesis" is additional function, which will be used by 1/20 or less users. so previous naming was quite precise.